src/EnergyPlus/PlantPressureSystem.cc

```diff
@@ -753,7 +743,7 @@ void PassPressureAcrossMixer(
 void PassPressureAcrossSplitter(EnergyPlusData &state,
                                 int const LoopNum,
                                 const DataPlant::LoopSideLocation LoopSideNum,
-                                Real64 &SplitterInletPressure)
+                                Real64 const SplitterInletPressure)
```

Comment:
Also revert reference to scalar.

[src/EnergyPlus/PlantPressureSystem.cc:728]:(style),[constParameter],Parameter 'MixerPressure' can be declared as reference to const
[src/EnergyPlus/PlantPressureSystem.cc:756]:(style),[constParameter],Parameter 'SplitterInletPressure' can be declared as reference to const